ghastly/ghastly.py

```diff
@@ -0,0 +1,292 @@
+import numpy as np
+import openmc
+import ghastly
+from jinja2 import Environment, PackageLoader
+from ghastly import read_input
+from ghastly import region
+from lammps import lammps
+
+env = Environment(loader=PackageLoader('ghastly','templates'))
+
+def fill_core(input_file, rough_pf):
+    '''
+    This function uses OpenMC's model.pack_spheres function to "rough" pack
+    the full core region defined by the Ghastly input file, then uses a series
+    of Jinja templates to create a LAMMPS input file that will run a physics
+    simulation, modeling not only the settling of the roughly packed pebbles
+    due to gravtiy, but also handle the addtion of any extra pebbles needed to
+    reach the target number of pebbles needed.
+
+    Parameters
+    ----------
+    input_file : str
+        Name of ghastly input file, which should be in JSON format, following
+        the layout and parameter naming convention given in the example inputs.
+    rough_pf : float
+        Packing fraction used in OpenMC packing function.  Can match the target
+        packing fraction provided in the input file, but users should take care 
+        supplying a rough_pf that is greater than the input's target pf.
```

Comment:
Does this mean users should supply a `rough_pf` that is greater, or that they should be careful if they do so?